rsts/deployment/aws/opta.rst

```diff
@@ -1,130 +1,155 @@
+Kubernetes cluster and node pool (with encrypted disk storage). And lastly, the k8s-base module sets up the resources
 within Kubernetes like the autoscaler, metrics server, and ingress.
 
-**Production Grade Database**
-The aws-postgres module in flyte.yaml creates an Aurora Postgresql database with disk encryption and regular snapshot
-backups. You can read more about it `here <https://docs.opta.dev/modules-reference/service-modules/aws/#postgres>`__
-
-**Production Grade Object Store**
-The aws-s3 module in flyte.yaml creates a new S3 bucket for Flyte, including disk encryption. You can read more about it
-`here <https://docs.opta.dev/modules-reference/service-modules/aws/#aws-s3>`__
-
-**Production Grade Notification System**
-Flyte uses a combination of the AWS Simple Notification Service (SNS) and Simple Queueing service for a notification
-system. flyte.yaml creates both the SNS topic and SQS queue (via the notifcationsQueue and topic modules), which are
-encrypted with unique KMS keys and only the  flyte roles can access them. You can read more about the queues
+Production-grade Database
+*************************
+The aws-postgres module in ``flyte.yaml`` creates an Aurora PostgreSQL database with disk encryption and regular snapshot
+backups. You can read more about it `here <https://docs.opta.dev/modules-reference/service-modules/aws/#postgres>`__.
+
+Production-grade Object Store
+*****************************
+The aws-s3 module in ``flyte.yaml`` creates a new S3 bucket for Flyte, including disk encryption. You can read more about it
+`here <https://docs.opta.dev/modules-reference/service-modules/aws/#aws-s3>`__.
+
+Production-grade Notification System
+************************************
+Flyte uses a combination of the AWS' Simple Notification Service (SNS) and Simple Queue Service (SQS) for the notification
+system. ``flyte.yaml`` creates both the SNS topic and SQS queue (via the notifcationsQueue and topic modules), which are
+encrypted with unique KMS keys and only the Flyte roles can access them. You can read more about the queues
 `here <https://docs.opta.dev/modules-reference/service-modules/aws/#aws-sqs>`__ and the topics
 `here <https://docs.opta.dev/modules-reference/service-modules/aws/#aws-sns>`__.
 
-**Production Grade Queueing System**
-Flyte uses SQS to power its task scheduling system, and flyte.yaml creates said queue (via the schedulesQueue
-module) with encryption and principle of least privilege rbac access like the other SQS queue above.
+Production-grade Queueing System
+********************************
+Flyte uses SQS to power its task scheduling system, and ``flyte.yaml`` creates said queue (via the schedulesQueue
+module) with encryption and principle of least privilege RBAC access like the SQS queue mentioned above.
 
-**Secure IAM Roles for Data and Control Planes**
+Secure IAM Roles for Data and Control Planes
+********************************************
```

Comment:
What do we have to write here? Currently, it's empty.

Reply:
Adding @wild-endeavor

Reply:
Added some content for now.